readme.txt

```diff
@@ -35,7 +35,8 @@ Do you prefer an eCommerce solution which is simple and intuitive or one which i
 Here are some of the features supported out-of-the-box:
 
 - **Frontend shopping cart** - Customers can browse your product catalogue, add items to their cart, and checkout when they are ready. Or they can bypass the cart using Buy Now buttons.
-- **Flexible and clean checkout form** - Buyers can complete their purchase with ease, directly on your site's checkout page.
+- **Highly customizable and clean checkout** - Buyers can complete their purchase with ease, directly on your site's checkout page.
+- **Payment flexibility** - Accept credit card payments using Stripe(https://stripe.com/) as well as PayPal payments. Apple Pay and Google Pay are also included.
```

Comment:
```suggestion
- **Payment flexibility** - Accept credit card payments using [Stripe](https://stripe.com/) as well as PayPal payments. Apple Pay and Google Pay are also included.
```

Reply:
With the ordering of the language here, it's not clear that Apple/Google Pay are included via Stripe.

Reply:
`Apple Pay and Google Pay are also included via Stripe's Payment Request Buttons`

or something similiar

Reply:
@cklosowski Agreed with your edit.

Reply:
Ok @pderksen I'll get it all committed.